staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -241,7 +241,6 @@ type CustomResourceSubresourceScale struct {
 	StatusReplicasPath string
 	// LabelSelectorPath defines the JSON path inside of a CustomResource that corresponds to Scale.Status.Selector.
```

Comment:
worth clarifying that it must be a JSON path to a string field (not a complex selector struct)? the examples given for workload types point to complex fields that would not work if they were embedded in a CRD and referenced by this field

Reply:
@liggitt Will it be more convenient if it is a JSON path to a selector struct? I filed an issue #69257 for it.

Reply:
@liggitt Will it be more convenient if it is a JSON path to selector struct? I filed an issue #69257 for it.